molgenis-app/dev-env/docker-compose.yml

```diff
@@ -1,11 +1,15 @@
 version: "3.4"
 services:
   frontend:
-    image: ${FRONTEND}
+    image: ${MOLGENIS_FRONTEND:-molgenis/molgenis-frontend:latest}
+    container_name: frontend
     ports:
       - 80:80
+    network_mode: ${NETWORK_MODE:-bridge}
     volumes:
-      - ${BACKEND}:/etc/nginx/proxy.d/backend.conf
+      - ./backend.conf:/etc/nginx/proxy.d/backend.conf
+    extra_hosts:
+      - host.docker.internal:${MOLGENIS_HOST:-172.17.0.1}
```

Comment:
This does not work for me on mac. If I remove this line it works fine.
I believe the MOLGENIS_HOST cannot be set reliably on mac based on what I find on https://docs.docker.com/docker-for-mac/networking/: 
> The host has a changing IP address (or none if you have no network access). From 18.03 onwards our recommendation is to connect to the special DNS name host.docker.internal, which resolves to the internal IP address used by the host.

Reply:
I shelled into the frontend container and asked it to resolve host.docker.internal. It resolves to 192.168.65.2 and if I set MOLGENIS_HOST=192.168.65.2 everything works.
However, I still suspect that this IP is random.

Reply:
I didn't like the extra_hosts option either. The whole reason for going that way was that docker-compose doesn't deal with variable interpolation in mounted volume config files(backend.conf) and that NGINX doesn't support config directives from the environment. I think I found an acceptable workaround that does the same with a little bit of bash magic. Instead of using a backend.conf, the file is being interpolated inside the container using envsubst and stored to backend.conf.